src/auro-popover.js

```diff
@@ -60,53 +74,95 @@ class AuroPopover extends LitElement {
 
   firstUpdated() {
     // needs to eval before Popover instantiation
-    let button = {};
+    let trigger = {};
 
     try {
       document.querySelector(`#${this.for}`).parentElement.appendChild(this.patchBuildless());
-      button = document.querySelector(`#${this.for}`);
+      trigger = document.querySelector(`#${this.for}`);
     } catch (err) {
       this.querySelector(`#${this.for}`).appendChild(this.patchBuildless());
-      button = this.querySelector(`#${this.for}`);
+      trigger = this.querySelector(`#${this.for}`);
     }
 
-    const element = this.shadowRoot.querySelector('#popover'),
-     hideEvents = [
-      'mouseleave',
-      'blur'
-      ],
-     popper = new Popover(button, element, this.placement),
-     showEvents = [
-      'mouseenter',
-      'focus'
-      ],
-
-    /**
-     * Hides the popover
-     * @returns {Void} Fires an update lifecycle.
-     */
-    toggleHide = function() {
-      element.removeAttribute('data-show');
-      popper.hide();
-    },
-
-    /**
-     * Shows the popover
-     * @returns {Void} Fires an update lifecycle.
-     */
-     toggleShow = function() {
-      element.setAttribute('data-show', '');
-      popper.show();
-    };
+    this.popover = this.shadowRoot.querySelector('#popover');
+    this.popper = new Popover(trigger, this.popover, this.placement);
+
+      /**
+      * Hides the popover
+      * @returns {Void} Fires an update lifecycle.
+      */
+      const toggleHide = () => {
+        this.popover.removeAttribute('data-show');
+        this.popper.hide();
+        this.isPopoverVisible = false;
+      },
+
+      /**
+       * Shows the popover
+       * @returns {Void} Fires an update lifecycle.
+       */
+      toggleShow = () => {
+        this.popover.setAttribute('data-show', '');
+        this.popper.show();
+        this.isPopoverVisible = true;
+      },
+
+      /**
+       * Click handler on non-trigger non-popovers
+       * @param {Event} event event
+       * @returns {Void} Fires an update lifecycle
+       */
+      handleClickNonTriggerNonPopover = (event) => {
+        const path = event.composedPath();
+
+        if (this.isPopoverVisible && !path.includes(trigger) && !path.includes(this.popover)) {
+          toggleHide();
+        }
+      },
+
+      /**
+       * Click handler focus-trigger
+       * @param {Event} event event
+       * @returns {Void} Fires an update lifecycle
+       */
+      handleTabWhenFocusOnTrigger = (event) => {
+        if (event.key.toLowerCase() === 'tab') {
+          toggleHide();
+        }
+      };
+
+    if (this.sticky) {
+      trigger.addEventListener('click', toggleShow);
+      trigger.addEventListener('focus', toggleShow);
```

Comment:
```suggestion

```

I am going to ask to remove this. As an accessibility attribute of the experience, things popping up when tabbed to them is not desired. Also, this is not consistent with the other popovers in the demo. 

The specification of the sticky popover is to have it stay open on a click versus a fly-by when hovered. 

Also, this update alerted me to the fact that our popovers are not really accessible. There is no way to tab into the scope of the popover. Also, the content within the popover is tabbable and ready aloud even when the popover is closed. 

These a11y issues will be addressed in another issue.